content/v2.0/router/overview.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Router: Overview"
+order: 10
+---
+
+# Overview
+
+Hanami applications use [Hanami::Router](https://github.com/hanami/router) for routing: a Rack compatible, lightweight and fast HTTP router for Ruby.
+
+## The first route
+
+With your favorite editor open `/config/routes.rb` and add the following line inside of the `main` slice block.
+
+```ruby
+get '/hello', to: ->(env) { [200, {}, ['Hello from Hanami!']] }
+```
+
+So the whole file would look like this:
+
+```ruby
+# /config/routes.rb
+
+Hanami.application.routes do
+  slice :main, at: "/" do
```

Comment:
Should we explain how slice routing works here too, at least briefly? There will be more guides about slices, of course, but this will be a new concept to almost everyone so briefly mentioning it's something new to learn could be useful (though the routing portion here is almost trivial and obvious).

Reply:
This is also a topic I briefly mentioned in the `basic_usage` - Let's discuss if it's better to put it here when I'll push remaining docs :). I've not a strong opinion about it yet.